app/routes/candidates.js

```diff
@@ -6,7 +6,7 @@ var db = new sqlite3.Database('db');
 /* GET users listing. */
 router.get('/', function(req, res, next) {
   var candidates = [];
-  var selectClause = 'SELECT firstName, lastName, party, chamber, state, district, incumbent, website, email, facebook, twitter, youtube FROM candidates';
+  var selectClause = 'SELECT img_src, firstName, lastName, party, chamber, state, district, incumbent, website, email, facebook, twitter, youtube FROM candidates';
```

Comment:
@terry77228 My last commit add the img_src as last column. Does the order here matters?